.github/workflows/pr-beta-release.yml

```diff
@@ -0,0 +1,309 @@
+name: PR Beta Release
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  issue_comment:
+    types: [created]
+
+jobs:
+  check-conditions:
+    if: github.event_name == 'issue_comment' && github.event.issue.pull_request
```

Comment:
aslo check that the comment from the repository owner
do not trigger flow if comment was posted another person